### ISSUE TITLE
[BUG] FastAI quickstart notebook failure

### ISSUE THREAD

**Issue by yueguoguo** [Microsoft/Recommenders#631]
### Description
It looks like the failure of fastai is owing to the train-valid split in model building. 

Cell 17 in the [notebook](https://github.com/Microsoft/Recommenders/blob/master/notebooks/00_quick_start/fastai_movielens.ipynb) creates a new test dataset that removed the seem items. 
```
training_removed = pd.concat([users_items, train_valid_df[[USER,ITEM]]]).drop_duplicates(keep=False)
```
However, `train_valid_df` also includes the user-item pairs in the validation set (not sure if there is a way to find out the user-item pairs used in the validation dataset in the model building). These items are not seen in the training set for building the model.

### In which platform does it happen?
Azure Data Science Virtual Machine

### How do we replicate the issue?
Clone the notebooks and run the fast ai [notebook](https://github.com/Microsoft/Recommenders/blob/master/notebooks/00_quick_start/fastai_movielens.ipynb)

### Expected behavior (i.e. solution)
It should run successfully

**Comment by miguelgfierro** [comment]
@gramhagen and I have been looking into this today. We found that in this old commit, things were working https://github.com/Microsoft/Recommenders/blob/benchmark_feb/notebooks/03_evaluate/comparison_movielens.ipynb

**Comment by gramhagen** [comment]
addressed in #646
